src/components/Trend/Trend.js

```diff
@@ -53,7 +53,7 @@ class Trend extends Component {
 
   componentDidMount() {
     const path = document.querySelector('.trend-line path');
```

Comment:
To make this a little bit easier. and not to break the compositional nature of react components. I think we can have this condition in the render method and add the class conditional.

The reason for this is using reference to DOM node breaks the compositional nature of react. We shouldn't use dom reference to mutate the nodes that are already being controlled by virtual DOM. that way it wouldn't be confused on what to update and what not. Does this make sense?